pkg/tagger/collectors/kubernetes_servicemapper_nocompile.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build !kubeapiserver
```

Comment:
do we  still need this now that it's a different collector?

Reply:
I removed it !